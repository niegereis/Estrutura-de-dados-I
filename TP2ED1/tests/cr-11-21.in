11 21
r
*********************
*   * *     *       *
* * * * *** * ***** *
* *   * * *   *M    *
* * *** * ***********
* * *   *           *
* * * *** ***** *** *
* * * *       * * * *
* *** ********* * * *
*               *    
*********************
