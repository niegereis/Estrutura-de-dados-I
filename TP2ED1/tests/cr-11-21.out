55
*********************
*   * *ooooo*ooooooo*
* * * *o***o*o*****o*
* *   *o* *ooo*Moooo*
* * ***o* ***********
* * *ooo*      ooooo*
* * *o*** *****o***o*
* * *o*       *o* *o*
* ***o*********o* *o*
*    ooooooooooo*  oo
*********************
